### PR TITLE
Login

### PR DESCRIPTION
**Pull request by ReynardEra**
Fixed background issues in Login_index.css and Login_index.js

### CODE REVIEW

File: src/Login/Login_index.js [hunk 1]
```diff
@@ -27,56 +27,54 @@ class App extends Component {
...
+								floatingLabelText="Email"
+								type="email"
+								onChange = {(event,newValue) => this.setState({email:newValue})}
+							/>
+							<br></br>
+							<TextField className="loginbox"
+								hintText="Password"
+								floatingLabelText="Password"
+								type="password"
+								onChange = {(event,newValue) => this.setState({password:newValue})}
```
Comment (KimberleighBeyl) [inline src/Login/Login_index.js]: Redux!!!!

Comment (ReynardEra) [reply src/Login/Login_index.js]: Ek dink ek het Redux in gesit vir validation.

Comment (KimberleighBeyl) [reply src/Login/Login_index.js]: Why setstate then?

**Comment by ReynardEra** [comment]
Have to do Redux

**Pull request closed without merging**